### PR TITLE
[angular] Update Angular README

### PR DESCRIPTION
**Pull request by wmarques**
As requested in #10624 

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

File: generators/common/templates/README.md.ejs [hunk 1]
```diff
@@ -200,6 +201,14 @@ The service worker initialization code is commented out by default. To enable it
 ```
 
 Note: [Workbox](https://developers.google.com/web/tools/workbox/) powers JHipster's service worker. It dynamically generates the `service-worker.js` file.
+<%_ } _%>
+<%_ if (clientFramework === ANGULAR) { _%>
+The service worker initialization code is disabled by default. To enable it, uncomment the following code in `src/main/webapp/app/app.module.ts`:
```
Comment (DanielFran) [inline generators/common/templates/README.md.ejs:206]: Change `src/main/webapp/app/app.module.ts` by `<%= CLIENT_MAIN_SRC_DIR %>webapp/app/app.module.ts`

**Review by DanielFran — CHANGES_REQUESTED** [review]

**Review by gmarziou — COMMENTED** [review]
Do you think that the Leaflet example in "Managing Dependencies" paragraph could/should be rewritten using `ng add`?

**Comment by wmarques** [comment]
@gmarziou I think it's better to keep it. Not all dependencies use Angular Schematics so let's keep the "standard" way, it's more generic IMO.

**Comment by gmarziou** [comment]
@wmarques OK it makes sense. 

I will test it soon and review the README by comparison with the one generated by Angular CLI, maybe I'll propose minor changes.

File: generators/common/templates/README.md.ejs (hunk 1 above)
Comment (wmarques) [reply generators/common/templates/README.md.ejs]: Done, thanks ! :)

**Comment by wmarques** [comment]
@gmarziou @DanielFran it should be ok, thanks for your comments! Let me know if you have any others

**Review by DanielFran — APPROVED** [review]
LGTM

**Pull request merged**